crates/duty-tracker/src/contract_manager.rs

```diff
@@ -171,16 +171,16 @@ impl ContractManagerCtx {
                     })
                     .collect::<Result<BTreeMap<OperatorIdx, PegOutGraphSummary>, TxGraphError>>()?;
                 let (sm, duty) = ContractSM::new(
-                    self.build_context.own_index(),
+                    todo!(), // TODO(proofofkeags): We need to get our own operator pubkey.
```

Comment:
This can be done by the Secret Service in https://github.com/alpenlabs/strata-bridge/blob/33bb8af6425369d23fa9ebf9e2acd1bdc1e0e35e/crates/secret-service-client/src/musig2.rs#L68-L77

@azz can you confirm that this is the operator pubkey that is used to identify him in the `OperatorPubkeyTable`?

Reply:
Uhh it's unlikely. We don't do MuSig for signing P2P messages, do we?

Reply:
Then you can do this: https://github.com/alpenlabs/strata-bridge/blob/33bb8af6425369d23fa9ebf9e2acd1bdc1e0e35e/crates/secret-service-client/src/p2p.rs#L31-L40

and create a new `strata-p2p-impl::Configuration`: https://github.com/alpenlabs/strata-bridge/blob/33bb8af6425369d23fa9ebf9e2acd1bdc1e0e35e/crates/p2p-impl/src/config.rs#L9-L34

and call bootstrap: https://github.com/alpenlabs/strata-bridge/blob/33bb8af6425369d23fa9ebf9e2acd1bdc1e0e35e/crates/p2p-impl/src/bootstrap.rs#L14

Reply:
I'll add an easy constructor to the `strata-p2p-impl::Configuration` that takes a `SecretKey` and also a method that returns an `XOnlyPublicKey`.